ui/app/services/csp-event.js

```diff
@@ -3,34 +3,35 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-/*eslint-disable no-constant-condition*/
-import { computed } from '@ember/object';
-
 import Service from '@ember/service';
+import { tracked } from '@glimmer/tracking';
 import { task, waitForEvent } from 'ember-concurrency';
+import { addToArray } from 'vault/helpers/add-to-array';
 
-export default Service.extend({
-  events: computed(function () {
-    return [];
-  }),
-  connectionViolations: computed('events.@each.violatedDirective', function () {
-    return this.events.some((e) => e.violatedDirective.startsWith('connect-src'));
-  }),
+export default class CspEventService extends Service {
+  @tracked connectionViolations = [];
```

Comment:
Full list of events didn't seem to be used anywhere, and the alias for the getter wasn't updating on the auth-form correctly so I refactored this service to track `connectionViolations` directly